suite-native/assets/src/components/Assets.tsx

```diff
@@ -63,8 +63,8 @@ export const Assets = () => {
 
     return (
         <>
-            <Card>
-                <VStack spacing={19}>
+            <Card noPadding>
+                <VStack spacing={0}>
```

Comment:
`VStack` with `spacing={0}` has no meaning. Replace this with a `Box`.

Reply:
Removed completely, `Card` is enough